FirebaseMLModelDownloader/Sources/CustomModel.swift

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+@objc(FIRCustomModelFormat) public enum CustomModelFormat : Int {
+  case Unknown
+  case TFLite
+  case TorchScript
+  case CoreML
+}
+
+@objc(FIRCustomModel) public class CustomModel : NSObject {
+
+  public let modelName: String
+  public internal(set) var modelSize: Int?
+  public internal(set) var modelPath: String?
+  public internal(set) var modelHash: String?
+  public internal(set) var modelFormat: CustomModelFormat?
+
+  init(withName name: String) {
+    modelName = name
+  }
+
+  public func getLatestModel() -> FileHandle? {
```

Comment:
What's the benefit of this function existing vs a developer just checking for themselves:

```
let customModel = someFuncToGetCustomModel()
if let modelPath = customModel.modelPath {
  // Use FileManager to open modelPath...
}
```

Reply:
I think the reasoning here is that the dev would probably have no use for modelPath itself, and it is easier to just provide them with the modelFile as needed. I've made the modelPath property internal as well. Would that work?